### PR TITLE
Adds basic Toolbar component tests.

### PR DESCRIPTION
**Pull request by BE-Webdesign**
Adds basic Toolbar component tests. Related to progress on #641.

**Testing Instructions**
Run npm i && npm run test-unit ensure tests pass. Change component logic
to ensure tests fail as they should.

### CODE REVIEW

File: components/toolbar/test/index.js [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+					title: 'WordPress',
+					subscript: 'wp',
+					onClick: clickHandler,
+					isActive: false,
+				},
+			];
+			const toolbar = shallow( <Toolbar controls={ controls } /> );
+			const listItem = toolbar.find( 'IconButton' );
+			expect( toolbar.type() ).to.equal( 'ul' );
+			expect( listItem.props().icon ).to.equal( 'wordpress' );
```
Comment (aduth) [inline components/toolbar/test/index.js]: Minor: `listItem.prop( 'icon' )` more explicit. Same goes for most of the following lines. Else we could test:

```js
expect( listItem.props() ).to.eql( {
	// ...keys
} );
```

File: components/toolbar/test/index.js [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+					title: 'WordPress',
+					subscript: 'wp',
+					onClick: clickHandler,
+					isActive: true,
+				},
+			];
+			const toolbar = shallow( <Toolbar controls={ controls } /> );
+			const listItem = toolbar.find( 'IconButton' );
+			listItem.simulate( 'click', { stopPropagation: () => undefined } );
+			expect( clicked ).to.be.true();
```
Comment (aduth) [inline components/toolbar/test/index.js]: Sinon spies are great for this sort of test:

http://sinonjs.org/releases/v2.3.2/spies/

Appears we're now using `sinon-test` as of #924, so it'd be matter of:

```js
it( 'should call the clickHandler on click.', sinon.test( function() {
	const clickHandler = this.spy();
	// ...
	expect( clickHandler ).to.have.been.calledOnce;
} );
```

See also: https://github.com/domenic/sinon-chai

**Review by aduth — APPROVED** [review]
Thanks for working on these tests 👍

File: components/toolbar/test/index.js (hunk 1 above)
Comment (BE-Webdesign) [inline components/toolbar/test/index.js]: Yeah, I thought I updated these, guess not lol. Will change.  Should it be changed to the one command style?

```
expect( listItem.props() ).to.eql( {
	// ...keys
} );
```

File: components/toolbar/test/index.js (hunk 2 above)
Comment (BE-Webdesign) [inline components/toolbar/test/index.js]: Yeah, I realized sinon was being used in our effects tests, so I will use that here as well in the revisions.

File: components/toolbar/test/index.js (hunk 1 above)
Comment (aduth) [inline components/toolbar/test/index.js]: I don't have a preference.

**Comment by BE-Webdesign** [comment]
@iseulde If you could take a peek at this to make sure this aligns with the changes you made, it would be appreciated.

**Pull request merged**